integration_tests/src/main/python/json_matrix_test.py

```diff
@@ -895,7 +906,8 @@ def test_from_json_doubles(std_input_path, input_file):
     pytest.param("boolean_formatted.json", marks=pytest.mark.xfail(reason='https://github.com/rapidsai/cudf/issues/15260')),
     pytest.param("int_array_formatted.json", marks=pytest.mark.xfail(reason='https://github.com/NVIDIA/spark-rapids/issues/10573')),
     "int_struct_formatted.json",
-    pytest.param("int_mixed_array_struct_formatted.json", marks=pytest.mark.xfail(reason='https://github.com/rapidsai/cudf/issues/15260'))])
+    pytest.param("int_mixed_array_struct_formatted.json", marks=pytest.mark.xfail(reason='https://github.com/rapidsai/cudf/issues/15260')),
+    pytest.param("escaped_strings.json", marks=pytest.mark.xfail(reason='https://github.com/rapidsai/cudf/issues/15260'))])
```

Comment:
just to double check, is this the right github issue for the escaped_strings xfail? Should it not be: https://github.com/NVIDIA/spark-rapids/issues/10196

Reply:
same comment for the ones below

Reply:
Yes it is. It does not look that way on the surface, but it is failing with an error

```
column_factories.hpp:343: Invalid, non-fixed-width type.
```

mixed_type_as_string means that it sees multiple different types and tries to return the value as a string. Here it looks like the values are already strings and it is the same error. If you want me to file a separate issue I can.

Reply:
No that's fine. Thanks